addons/data_recycle/models/data_recycle_model.py

```diff
@@ -174,15 +174,15 @@ def _send_notification(self, delta):
         ])
         partner_ids = self.notify_user_ids.partner_id.ids if records_count else []
         if partner_ids:
-            menu_id = self.env.ref('data_recycle.menu_data_cleaning_root').id
+            menu_id = self.env.ref('data_recycle.menu_data_cleaning_root', raise_if_not_found=False)
             self.env['mail.thread'].message_notify(
                 body=self.env['ir.qweb']._render(
                     'data_recycle.notification',
                     {
                         'records_count': records_count,
                         'res_model_label': self.res_model_id.name,
                         'recycle_model_id': self.id,
-                        'menu_id': menu_id
+                        'menu_id': menu_id and menu_id.id or False
```

Comment:
If we pass `False`, then "here" link won't work

https://github.com/odoo/odoo/blob/92442878b798b823b37b82dfee773b9001d159fc/addons/data_recycle/views/data_recycle_templates.xml#L5

Reply:
> If we pass `False`, then "here" link won't work
> 
> https://github.com/odoo/odoo/blob/92442878b798b823b37b82dfee773b9001d159fc/addons/data_recycle/views/data_recycle_templates.xml#L5

Hii @yelizariev I fixed these changes as you requested. Please check it and let me know if you have any other suggestions.